zebrad/tests/acceptance.rs

```diff
@@ -1706,6 +1706,57 @@ async fn delete_old_databases() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn stored_config_works() -> Result<()> {
```

Comment:
The ports and paths in this test will conflict with valid_generated_config.

To avoid the conflict, call this function in valid_generated_config_test after valid_generated_config finishes.

Reply:
Not needed if our tests are not async ?

Reply:
`cargo test` runs tests in parallel in different threads:
https://doc.rust-lang.org/cargo/commands/cargo-test.html#description

If the tests are async, they get launched on the async executor, but either way they run concurrently.